build/commands/lib/test.js

```diff
@@ -224,6 +233,7 @@ const runTests = (passthroughArgs, suite, buildConfig, options) => {
       }
       if (
         config.targetOS === 'android'
+        && !suite.endsWith('_junit_tests')
```

Comment:
nit: this `endsWith()` call now repeats everywhere. I'd create a var `const isJunitTestSuite = suite.endsWith('_junit_tests')` and use it everywhere to have less error-prone code.